src/index.ts

```diff
@@ -1,12 +1,51 @@
+import express, { NextFunction, Request, Response } from 'express';
+import helmet from 'helmet';
+// @ts-ignore
+import xss from 'xss-clean';
+import ExpressMongoSanitize from 'express-mongo-sanitize';
+import morgan from 'morgan';
+import { errorHandler } from './modules/errors/errorHandler.ts';
+import { AppError } from './modules/errors/AppError.ts';
+
 import mongoose from 'mongoose';
 import config from './config/config.ts';
-import { app } from './app.ts';
+import 'reflect-metadata';
+import { InversifyExpressServer } from 'inversify-express-utils';
+import { Container } from 'inversify';
+import { UserService } from './modules/user/user.service.ts';
+import './modules/user/user.controller.ts';
+import TYPES from './constant/types.ts';
 
 let server: any;
 
 mongoose.connect(config.mongoose.url!).then(() => {
   console.log('DB connection successful!');
-  server = app.listen(config.port, () => {
+
+  const container = new Container();
+
+  container.bind<UserService>(TYPES.UserService).to(UserService);
```

Comment:
It's better to put container creation and all bindings into a separate config file like shown in the inversify documentation here: https://www.npmjs.com/package/inversify#step-3-create-and-configure-a-container